api/build.gradle

```diff
@@ -152,5 +152,5 @@ pitest {
     outputFormats = ['XML', 'HTML']
     historyInputLocation = 'build/reports/pitest/fastermutationtesting'
     historyOutputLocation = 'build/reports/pitest/fastermutationtestingoutput'
-    mutationThreshold = 90
+    mutationThreshold = 40
```

Comment:
@Jennings-HMCTS - I hope we have a separate ticket created to look into the threshsold issue and it is a temporary fix to make build green.

Reply:
This is not a fix.